bigquery/cloud-client/authorized_view_tutorial_test.py

```diff
@@ -23,24 +23,21 @@ def client():
     return bigquery.Client()
 
 
-@pytest.fixture
-def to_delete(client):
-    doomed = []
-    yield doomed
-    for item in doomed:
-        if isinstance(item, (bigquery.Dataset, bigquery.DatasetReference)):
-            client.delete_dataset(item, delete_contents=True)
-        elif isinstance(item, (bigquery.Table, bigquery.TableReference)):
-            client.delete_table(item)
-        else:
-            item.delete()
-
-
-def test_authorized_view_tutorial(client, to_delete):
+def remove_authz_views(client, dataset_ref):
+    try:
+        ds = client.get_dataset(dataset_ref)
+        ds.access_entries = [entry for entry in ds.access_entries
+                             if entry.entity_type != 'view']
+        client.update_dataset(ds, ['access_entries'])
+    except Exception:
+        pass
```

Comment:
nit: can we add a print statement at minimum here or catch a specific exception?